### PR TITLE
Add Languages to Filter

### LINKED ISSUE #2488: Add languages to the technologies project filter
### Dependency
#2487

### Overview
The languages for each project under the projects page should be included as a filter with the Technologies

### Action Items
- [x] Become familiar with the JS in the [project.html](https://github.com/hackforla/website/blob/gh-pages/_layouts/project.html) and how it obtains the languages used for a project from [_data/external/github-data.json](https://github.com/hackforla/website/blob/gh-pages/_data/external/github-data.json)
- [x] In the [projects page](https://www.hackforla.org/projects/), update each project page card so the languages are listed
    - [x] You should be able to utilize the same JS as above to get the languages for a project
    - [x] Each project card and the contents inside are generated in the [current-projects.js](https://github.com/hackforla/website/blob/gh-pages/assets/js/current-projects.js) file
    - [x] You can put the 'Languages' field in between 'Partner' and 'Technologies'
    - [x] Refer to the screenshots below
- [x] In the [projects page](https://www.hackforla.org/projects/), update the filter so projects can be filtered by Technologies and Languages
    - [x] Under the existing 'Technologies' filter, add the languages from the projects
    - [x] Change the name of the filter to 'Languages / Technologies'
    - [x] The filter should be able to filter by languages and/or technologies
    - [x] Refer to the [current-projects.js](https://github.com/hackforla/website/blob/gh-pages/assets/js/current-projects.js) file
- [x] Make sure the page looks fine for smaller viewport screens as well, such as tablet, mobile (responsiveness)
- [x] Once the PR is completed and merged, release dependency on #2502

**Screenshots**
<details>
<summary>Languages from here</summary>

c1

</details>

<details>
<summary>Should appear here</summary>

c2

</details>

### Resources/Instructions
[projects page](https://www.hackforla.org/projects/)
[current-projects.js](https://github.com/hackforla/website/blob/gh-pages/assets/js/current-projects.js)

### PR DESCRIPTION
**Pull request by sydneywalcoff**
Fixes #2488

### What changes did you make and why did you make them ?

  - added the applicable languages to the filter using the github data
  - changed the filter title to be " languages / technologies"
  - added the languages to data-technologies attribute

### Screenshots of Proposed Changes Of The Website  (if any, please do not screen shot code changes)

<details>
<summary>Visuals before changes are applied</summary>

screenshot showing all cards because language filter not possible and language not displayed on card

</details>

<details>
<summary>Visuals after changes are applied</summary>
  
screenshot showing filtering for 'CSS' and languages displayed on each project card

</details>

### CODE REVIEW

**Comment by SAUMILDHANKAR** [comment]
Review ETA: EOD 1/5/22
Availability: 6 hrs

**Comment by sydneywalcoff** [comment]
I checked mobile responsiveness and adding the languages to the dropdown makes it extend past the footer into white space. Should we kick this back to design or just add a scroll bar?

**Comment by ExperimentsInHonesty** [comment]
@sydneywalcoff please add a scroll bar so that we can push this out.  Literally tonight someone asked if there is a project that uses python and they could not search the site.  We can always make it look pretty later.

**Comment by SAUMILDHANKAR** [comment]
Dropdown extends past the footer into white space in current site as well. So, that can be fixed in a separate issue
filter extending into whitespace on mobile view (current site)
.

File: assets/js/current-projects.js [hunk 1]
```diff
@@ -19,15 +19,17 @@ document.addEventListener("DOMContentLoaded",function(){
         let filters = createFilter(sortedProjectData);
         
         // Insert Checkbox Filter Into The Dom
-        for(const [filtername,filtervalue] of Object.entries(filters)){
+        for(const [filterName,filterValue] of Object.entries(filters)){
             // Add displayed filter title, resolves issue of "program areas" not being valid html attribute name due to spacing
```
Comment (SAUMILDHANKAR) [inline assets/js/current-projects.js:23]: @sydneywalcoff: Thanks for working on this issue. I am able to see the languages on the project page as well as the filter. Great job! Just a minor note, is this comment still required?

**Comment by SAUMILDHANKAR** [comment]
@sydneywalcoff I see the filter name as Technologies when clicking on Python. Not a major issue but just wondering if it could be changed to languages /technologies or better yet languages?
Technology filter

File: assets/js/current-projects.js (hunk 1 above)
Comment (sydneywalcoff) [reply assets/js/current-projects.js:23]: @SAUMILDHANKAR Do you mean the comment on line 21 or line 23?

**Comment by sydneywalcoff** [comment]
@SAUMILDHANKAR 

> @sydneywalcoff I see the filter name as Technologies when clicking on Python. Not a major issue but just wondering if it could be changed to languages /technologies or better yet languages? Technology filter

The reason I did this was because of the comment on line 23; 'languages / technologies' would not be an appropriate html attribute because of the spaces. They did something similar with 'Program Areas'. It displays as 'Programs'. I figured that 'Technologies' was a more inclusive term than 'Languages'.

**Review by R-Tomas-Gonzalez — APPROVED** [review]
Seems like this needs to be taken care of asap. I have created some notes for this, but will save for Sunday meeting. 

Thanks for you work @sydneywalcoff.

Cheers,

- Tomas

File: assets/js/current-projects.js (hunk 1 above)
Comment (R-Tomas-Gonzalez) [reply assets/js/current-projects.js:23]: @sydneywalcoff @SAUMILDHANKAR Lets leave the comments there. Sydney, I would say let's push this out so we can have this feature. But from here, we can create another issue that is concerning the other things that need to be taken care of. Thanks!

**Pull request merged**